data/campaigns/Under_the_Burning_Suns/units/quenoth/Mystic.cfg

```diff
@@ -7,17 +7,19 @@
     gender=female
     image=units/quenoth/mystic/mystic.png
     profile="portraits/quenoth/mystic.webp"
-    hitpoints=25
+    hitpoints=22
     movement_type=quenoth_foot
     movement=5
-    experience=28
+    experience=31
     level=1
-    alignment=neutral
+    alignment=lawful
```

Comment:
I forgot but why are we changing the alignment?

Reply:
Because every single unit in this line / race other than them is lawful.